client/deis.py

```diff
@@ -941,6 +942,36 @@ def auth_whoami(self, args):
             self._logger.info(
                 'Not logged in. Use `deis login` or `deis register` to get started.')
 
+    def auth_regenerate(self, args):
+        """
+        Regenerates auth token, defaults to regenerating token for the current user.
+
+        Usage: deis auth:regenerate [options]
+
+        Options:
+          -u --username=<username>
+            specify user to regenerate. Requires admin privilages.
+          --all=<all>
```

Comment:
We should change this to `--all` so it'll end up being true if set, otherwise users will have to use `--all=foo` in order to use this feature flag.